### PR TITLE
Update chartmuseum

### PR DESCRIPTION
**Pull request by faizanahmad055**
Signed-off-by: Faizan Ahmad
username: faizanahmad055
email: faizan.ahmad55@outlook.com

#### What this PR does / why we need it:
- Add support for match labels for deployment selector
- Add support for additional labels in deployment
- Add support for volume in pvc
- Add pv in chartmuseum

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by faizanahmad055** [comment]
Hi @jdolitsky, @goruha
Can you guys please take a look.

File: stable/chartmuseum/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Helm Chart Repository with support for Amazon S3 and Google Cloud Storage
 name: chartmuseum
-version: 1.6.2
+version: 1.7.2
```
Comment (jdolitsky) [inline stable/chartmuseum/Chart.yaml]: Please use `1.7.0`

File: stable/chartmuseum/README.md [hunk 2]
```diff
@@ -102,6 +110,7 @@ their default values. See values.yaml for all available options.
 | `env.open.CACHE`                       | Cache store, can be one of: redis           | ``                                                  |
 | `env.open.CACHE_REDIS_ADDR`            | Address of Redis service (host:port)        | ``                                                  |
 | `env.open.CACHE_REDIS_DB`              | Redis database to be selected after connect | `0`                                                 |
+| `env.field.KUBERNETES_NAMESPACE`       | Namespace name in which chartmuseum is deployed | ``                                              |
```
Comment (jdolitsky) [inline stable/chartmuseum/README.md]: Should not be necessary to have this field, can just use  `{{ .Release.Namespace }}`

File: stable/chartmuseum/templates/deployment.yaml [hunk 3]
```diff
@@ -31,6 +42,14 @@ spec:
           value: {{ $value | quote }}
 {{- end }}
 {{- end }}
+{{- range $name, $value := .Values.env.field }}
+{{- if not ( empty $value) }}
+        - name: {{ $name | quote }}
+          valueFrom:
+            fieldRef:
+              fieldPath: {{ $value | quote }}
```
Comment (jdolitsky) [inline stable/chartmuseum/templates/deployment.yaml:50]: Can you explain what this does / how its used?

**Review by jdolitsky — CHANGES_REQUESTED** [review]

File: stable/chartmuseum/templates/deployment.yaml (hunk 3 above)
Comment (faizanahmad055) [reply stable/chartmuseum/templates/deployment.yaml:50]: It is used to expose pod information to containers through environment variables. For example, POD_IP environment variable gets its value from the Pod’s status.podIP field
```yaml
- name: POD_IP
  valueFrom:
    fieldRef:
      fieldPath: status.podIP
```

File: stable/chartmuseum/README.md (hunk 2 above)
Comment (faizanahmad055) [reply stable/chartmuseum/README.md]: This was just an example of how to use this. But I have updated it anyway.

File: stable/chartmuseum/Chart.yaml (hunk 1 above)
Comment (faizanahmad055) [reply stable/chartmuseum/Chart.yaml]: Updated

**Comment by faizanahmad055** [comment]
Hi @jdolitsky and @goruha,
Ping

**Comment by faizanahmad055** [comment]
Hi @jdolitsky and @goruha,
Ping

**Review by jdolitsky — APPROVED** [review]

**Pull request merged**